notebooks/tfx_pipelines/walkthrough/solutions/tfx_walkthrough_vertex.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    )
</pre><p>.with_id("model_evaluator"). My thinking on the naming convention is to distinguish the Data components vs. the Model components from the Trainer onward.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
<p>Got you! Done!</p>